cli/deployment/config.go

```diff
@@ -359,6 +359,14 @@ func newConfig() *codersdk.DeploymentConfig {
 			Flag:       "user-workspace-quota",
 			Enterprise: true,
 		},
+		Provisionerd: &codersdk.ProvisionerdConfig{
```

Comment:
We have another config called `ProvisionerDaemons`. I think it would be sensible to unify these under `Provisioner.Daemons` and `Provisioner.ForceCancelInterval`?

Should be a non-breaking change as well since the flag/env translate to the same.

Reply:
I assumed that this would be a breaking change if there is a customer who uses the `ProvisionerDaemons`. I'm for unification, but a bit concerned about distributing somebody's environment.

Reply:
I think it should be fine. Neither flag or env should break (was `PROVISIONER_DAEMONS` and will be the same in the new structure, please correct me if I’m wrong @f0ssel).

The only breakage I can think if is when used as part of `server.yaml`. But it’s a recent addition and not widely used yet AFAIK.

There may be even more breaking changes to this in the future as we split out provisionerd servers from the coderd server.